simulation/sub8_gazebo/worlds/contact.world

```diff
@@ -84,14 +91,18 @@
           </geometry>
         </collision>
       </link>
-    </model>
+    </model> -->
+    <include>
```

Comment:
Side note, when I run the full simulator on my laptop, it runs at like .2 times normal speed which seems to mess up some component of the subs navigation.